### PR TITLE
Adds draft of metrics documentation.

### PR DESCRIPTION
**Pull request by chuckharmston**
Note: the link to the Test Pilot standards will 404 while in review; for now, it can be read at https://github.com/mozilla/testpilot/blob/2538-ga-docs/docs/experiments/ga.md.

### CODE REVIEW

**Comment by chuckharmston** [comment]
CC @johngruen, @clouserw

**Comment by wresuolc** [comment]
This also looks fine to me.  @garethcull -- anything jump out at you?

File: docs/metrics.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- **Suggestion** – a single suggested string of text, as returned from the speech-to-text engine. Has a paired confidence level.
+
+## Analysis
+Data collected by Voice Fill will be used to answer the following high-level questions:
+
+- Is the Voice Fill experience compelling?
+	- How often do users reject suggestions and try again?
+	- How often do users accept and modify suggestions?
+	- What are users’ tolerance for inaccuracy?
+- Where to users use Voice Fill?
```
Comment (wresuolc) [inline docs/metrics.md]: s/to/do/

**Comment by wresuolc** [comment]
Thanks Chuck

**Pull request merged**